pages/research_software_stories/baler_research_software_story.md

```diff
@@ -0,0 +1,168 @@
+astronomers can apply it to reduce streaming telescope data in real time
+before transmission or storage.
+
+## Tools
+
+- GitHub Actions: Automates continuous integration and testing for every
+  update.
+
+- Black: Automatically formats the code to enforce a consistent style.
+
+- Docker: Ensures reproducible runtime environments for testing and
+  deployment.
+
+Using these tools improves the software's reliability and maintainability,
+making it easier for contributors to collaborate and for users to deploy
+BALER.
+
+## FAIR & Open
+
+BALER aligns with FAIR principles.  The source code is openly available on
+GitHub under the Apache 2.0 license, enabling broad access and reuse.  Each
+major release is archived on Zenodo with a DOI, which improves its
+findability and allows precise citation in publications.
+
+The team evaluated BALER against a FAIR software checklist and scored well
+on open licensing and accessibility.  They identified some areas to improve,
+such as refining versioning practices and making outputs more interoperable
+(currently, compressed outputs are NumPy arrays that require the
+corresponding ML model to decode).
+
+The BALER developers also engage in the broader community on software
+sustainability and FAIR practices.  For instance, they participate in an
+ESCAPE working group focused on sustainable and FAIR machine learning tools.
+
+## Documentation
+
+BALER's documentation is primarily hosted on its GitHub repository and wiki. 
+The main README provides instructions for installation and basic usage.
+
+The team also archives key documents (research papers, presentations,
+internal design reports) on Zenodo, and the project wiki links to these
+resources.  By layering the documentation (README, tutorials, wiki, archived
+reports), BALER ensures that knowledge is preserved despite frequent team
+turnover.  This makes it easier for new users and developers to learn the
+tool and contribute.
+
+## Sustainability
+
+The BALER team emphasises sustainability by preserving knowledge and
+following structured processes.  All code is under version control (Git),
+and major versions are archived on Zenodo for long-term access and
+reproducibility.  Documentation and design notes are routinely published
+online so that information remains available over time.
+
+Project governance includes oversight by postdoctoral researchers with
+guidance from senior academic staff, which provides stability and a
+long-term vision.  Since many contributors are students who may only work on
+BALER for a short period, the team aligns development goals with the
+academic calendar to maintain steady progress.
+
+Long-term funding, as always, is a challenge.  BALER currently relies on a
+variety of short-term project grants and volunteer effort.  By integrating
+BALER into larger initiatives, the team aims to secure more sustainable
+resources and support for continued development.
+
```

Comment:
I am just wondering about the timelessness of these comments on sustainability - a lot of the rest of the article is pretty timeless - especially if it links out to relevant parts of the project, RSQKit and other resources (and we will get to know about broken links when the linkchecker is fully up and running) - but things around sustainability might change so I am wondering if it's worth putting in a status date here - e.g.
"This is the situation as of July 2025 and funding and sustainability can and does change and this is here as a note on being open about challenges and not the ultimate source of information for the project/tool" <-- or something like that.